commons/tools/metrics.md

```diff
@@ -115,6 +115,287 @@ In the following example, we get the `accessLogs` status codes and create a GTS
 
 <script src="https://gist.github.com/cnivolle/2ee8607d995daa1316e17ffc3874d047.js"></script>
 
+## Monitoring' metrics
+
+All applications and VMs instances behind are monitored. Data is sent to [Warp10](/doc/tools/warp10/), a Geotimes 
+series database. 
+All metrics can be processed directly in the console in the Metrics tab of your applications or by the Clever Cloud 
+Warp10 endpoint.
+
+### Monitoring data model
+
+All metrics data follow the same schema in warp10. 
+Each class represents a specific metric. The context is provided by the warp10 labels.
+
+#### Class values and Labels
+
+##### Overview
+
+A telegraf daemon supplies most metrics.
+
+Each metric is recorded as a warp10 class. 
+Labels provide additional information about the VMs like instances id, organisation id, reverse proxy used.
+
+##### Labels
+
+In metrics' data, mains labels would be :
+
+- `owner_id` : A unique ID by organisation
+- `app_id` : A unique ID of application
+- `host` : HV id hosting the VM instance
+- `adc` : Reverse proxy ID
+- `vm_type` : `volatile` or `persistent`. Is it a stateless application or a stateful add-on
+- `deployment_id` : ID of the deployment
+
+> Depending on the class. Only some of this labels could be present
+
+##### Classes
+
+Telegraf provide lots of metrics described in their [documentation](https://github.com/influxdata/telegraf/tree/master/plugins/inputs).
+
+Below, the list of all warp10 classes representing Telegraf metrics :
+
+<table align="center">
```

Comment:
Please add `class="table table-bordered"` in the `<table>`. That will make the table easier to read.